components/html-block/demo/html-block.html

```diff
@@ -39,7 +39,7 @@
 		</script>
 		<script>
 			if (window.location.search.indexOf('latex=true') !== -1) {
-				document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({ renderLaTeX: true });
+				document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({ renderLatex: true });
```

Comment:
I noticed the html block demo wasn't working when `latex=true` was used. Looks like https://github.com/BrightspaceUI/core/blob/main/helpers/mathjax.js#L5 was updated to use `renderLatex` so this just makes that consistent.